src/ep_worker.c

```diff
@@ -9848,18 +9854,39 @@ static ep_stat_t w_form_query_getall_insert_new(worker_data_t *wd, obj_info_t *o
         strcat_safe(query, "], ", query_size);
     }
     /* The last index is calculated in the object's table. */
-    if (idx_params_num > 1)
-        strcat_safe(query, "IFNULL(MAX(t0.", query_size);
+     if (BackendIndex == -1 || subst_other_obj || idx_params_num > 1)
```

Comment:
What is it? I suggested to use this condition around FROM instead of hack with `MAX` to select something from empty table. It useless in this case.

Reply:
fixed